internal/admin/settings.go

```diff
@@ -0,0 +1,80 @@
+package admin
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"Twitter_like_application/config"
+	"Twitter_like_application/internal/services"
+
+	"github.com/gorilla/mux"
+)
+
+type SettingRequest struct {
+	Text string `json:"message"`
+}
+
+func SettingTweetLength(w http.ResponseWriter, r *http.Request, c config.Config) {
+	newLength, err := services.StrToInt(mux.Vars(r)["new_length"])
+	if err != nil {
+		services.ReturnErr(w, err.Error(), http.StatusInternalServerError)
+	}
+	c.MaxLengthTweet = strconv.Itoa(newLength)
+	if err != nil {
+		services.ReturnErr(w, err.Error(), http.StatusInternalServerError)
+	}
+	err = changeENV(newLength)
+	if err != nil {
+		services.ReturnErr(w, err.Error(), http.StatusInternalServerError)
+	}
+	m := SettingRequest{
+		Text: fmt.Sprintf("maximum tweet length is %d now", newLength),
+	}
+	services.ReturnJSON(w, http.StatusOK, m)
+}
+
+func changeENV(newLength int) error {
+	file, err := os.OpenFile(".env", os.O_RDWR, 0644)
```

Comment:
saving it in .env will not make any difference for the cases when this env var is set elsewhere. Either use a db or a different file (.cfg/.ini/.yaml/etc.)